src/App.css

```diff
@@ -313,6 +313,13 @@ form > span > .row > label {
   width: 100%;
 }
 
+iframe {
```

Comment:
This is kind of "the nuclear option," i.e. styling every instance of the HTML tag, as opposed to just the one instance. However, I'll approve it, as I'm going to guess that we _will_ want this for all `<iframe>` tag instances. Approved.